composer.json

```diff
@@ -20,26 +20,33 @@
         }
     },
     "require": {
-        "php": "^7.4|^8.0",
-        "react/socket": "^0.8",
-        "react/datagram": "^1.2",
-        "graylog2/gelf-php": "^1.5",
-        "symfony/process": "^4.1|^5.4",
-        "ext-json": "*"
+        "php": "^8.0",
+        "react/socket": "^1.12",
+        "react/datagram": "^1.9",
+        "graylog2/gelf-php": "^1.5|2.0",
```

Comment:
Tak doufejme, ze je to fakt kompatibilni :smile:

Reply:
to je to _file -> file, a UDP transfer neencoded message, takže to kompatibilní není a 1.5 vyhazuju